packages/circuit-ui/components/Card/Card.tsx

```diff
@@ -54,6 +54,6 @@ const spacingStyles = ({ theme, spacing = 'giga' }: StyleProps & CardProps) => {
   `;
 };
 
-export const Card: FC<CardProps> = styled('div', {
+export const Card: FC<PropsWithChildren<CardProps>> = styled('div', {
```

Comment:
Is this type annotation necessary at all? The types should be inferred automatically from the styled component. 

The use of the `FC` type is [discouraged](https://github.com/facebook/create-react-app/pull/8177).

Reply:
Nice, you're right, it does get inferred from `styled` 💯 will see if I can clean this up in other places as well

Reply:
Done in [88d33cf](https://github.com/sumup-oss/circuit-ui/pull/1554/commits/88d33cf91563faef62dd0a121ff61dd860d38d22)